app/cdap/text/text-en.yaml

```diff
@@ -3427,6 +3427,8 @@ features:
     enableQuotedValues: Enable quoted values
     encoding: Encoding
     format: Format
+    transformationErrorMessage1: Selected Transformation Cannot Be Applied
```

Comment:
add translations in features --> WranglerNewUI

Reply:
done